modules/cert-manager-operator-update-channels.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-operator-update-channels_{context}"]
+= Understanding update channels of the {cert-manager-operator}
```

Comment:
Attributes render in module titles is usually fine but have any other modules used this format and proved to now impact how titles render on the customer portal?

Reply:
Yes this is the published change https://docs.openshift.com/container-platform/4.13/security/cert_manager_operator/cert-manager-operator-install.html